_opensearch/query-dsl/full-text.md

```diff
@@ -192,7 +213,9 @@ GET _search
 }
 ```
 
-## Match phrase prefix
+### Match phrase prefix
+
+Use the `match_phrase_prefix` query type to specify a sequence of terms with a specified prefix. The documents that contain the phrase you specify will be returned. The last term in the phrase is partially provided as a prefix, so any documents that contain phrases that begin with the prefix for the last term will be returned. 
```

Comment:
Does this suggestion capture the meaning?
"The last partial term in the phrase is interpreted as a prefix, so any documents that contain phrases that begin with the phrase and prefix on the last term will be returned."

Reply:
thanks!